extensions/podman/src/podman-install.ts

```diff
@@ -305,9 +306,13 @@ class MacOSInstaller extends BaseInstaller {
       const pkgPath = path.resolve(getAssetsFolder(), `podman-installer-macos-${pkgArch}-v${podmanTool.version}.pkg`);
       try {
         if (fs.existsSync(pkgPath)) {
-          const runResult = await runCliCommand('open', [pkgPath, '-W']);
-          if (runResult.exitCode !== 0) {
+          try {
+            await extensionApi.process.exec('open', [pkgPath, '-W']);
+          } catch (err) {
             throw new Error(runResult.stdErr);
+            if ((err as extensionApi.RunError) && err.exitCode) {
```

Comment:
Don't know why we need to check for the exitCode as if we're inside the catch block it means exit Code was != 0